rspotify-model/src/playlist.rs

```diff
@@ -34,16 +32,13 @@ pub struct SimplifiedPlaylist {
     pub collaborative: bool,
     pub external_urls: HashMap<String, String>,
     pub href: String,
-    pub id: String,
+    pub id: PlaylistId,
```

Comment:
It seems these `Id` types have not implemented the customized serialize and deserialize function, is it ok to replace `String` with `PlaylistId`, could `serde` handle the serialization/deserialization of these `Id` type properly?

It's same to the other `Id` type.

Reply:
Ooh it's probably wrong now that you mention it, as the deserialization may even accept an uri on an invalid id. I'll take a look later on.

Reply:
This should be fixed after the last commit